ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -141,6 +161,7 @@ void RSkComponentTextInput::OnPaint(SkCanvas *canvas) {
   drawShadow(canvas, frame, borderMetrics, textInputProps.backgroundColor, layer()->shadowOpacity, layer()->shadowFilter);
   drawTextInput(canvas, component.layoutMetrics, textInputProps, textLayout);
   drawBorder(canvas, frame, borderMetrics, textInputProps.backgroundColor);
+  drawUnderline(canvas,frame,textInputProps.underlineColorAndroid);
```

Comment:
should this underline be before draw border ?

Reply:
agreed & addressed